packages/website/src/pages/index.tsx

```diff
@@ -126,8 +126,6 @@ function Sponsors(props: {
   title: string;
   className?: string;
 }): JSX.Element {
-  // TODO this seems like a ts-eslint problem: JSON types are not resolved
-  /* eslint-disable @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-call */
```

Comment:
This is... interesting. Didn't test it locally, but is that basically just because we used a local ESLint config instead of the global one? Is this actually a valid bug though that we can't resolve JSON types?

Reply:
Yeah I haven't dug in too deep. It's better now 🤷 

It would be nice to know but I don't quite have the time to poke too closely, heh.